imports/client/ui/components/timeLine/TimeLine.jsx

```diff
@@ -42,64 +42,65 @@ export default class TimeLine extends React.Component {
 
   openMaxDatePicker = () => {
     this.refs.maxDatePicker.focus()
-    //console.log('maxDAte')
+
   }
-PauseOrClear = (event ) => {
 
+  pauseOrClear = (event ) => {
 
-   window.clearInterval(this.tid);
-  if (this.playOrResumeTrig == 1){
-    //if paused before :clean
-    if (this.PauseOrClearTrig ==1) {
-      k = Math.round(this.props.ui.minTime)
-      this.props.updateUI({currentSliderTime :  k })
-      this.PauseOrClearTrig= 0
-      this.playOrResumeTrig =0
-      }
-    else {
 
-      this.PauseOrClearTrig= 1
+     window.clearInterval(this.timerForPlay);
+    if (this.playOrResumeTrig == 1){
```

Comment:
To stick with the general pattern of the app, it is better to use React's `state` property to manage the state of the component. More explanations here : https://egghead.io/lessons/react-state-basic